scripts/tests/test_render_platform_manifests.py

```
@@ -0,0 +1,125 @@
+    assert inputs.cluster_name == "cli", "CLI override should win"
+
+
+def test_build_render_tfvars_skips_vault_when_disabled(tmp_path: Path) -> None:
+    inputs = RenderInputs(
+        cluster_name="preview",
+        domain="example.test",
+        acme_email="admin@example.test",
+        cloudflare_api_token_secret_name=_dummy_secret_name(),
+        vault_address="https://vault.example",
+        vault_role_id="role",
+        vault_secret_id="secret",
```

Ruff (0.14.13)

Possible hardcoded password assigned to argument: "vault_secret_id"

(S106)